build.gradle

```diff
@@ -83,56 +87,56 @@ ospackage {
 
 def distributions = [
   [
-    "dist":        'openhab2-release',
-    "description": 'Linux installation package for openHAB 2.',
+    "dist":        'openhab-release',
+    "description": 'Linux installation package for openHAB.',
     "debDist":     ['stable','testing','unstable'],
-    "packageName": 'openhab2',
-    "url":         "https://bintray.com/openhab/mvn/download_file?file_path=org%2Fopenhab%2Fdistro%2Fopenhab%2F${OHVersion}%2Fopenhab-${OHVersion}.tar.gz",
+    "packageName": 'openhab',
+    "url":         "https://bintray.com/openhab/mvn/download_file?file_path=org/openhab/distro/openhab/${OHVersion}/openhab-${OHVersion}.tar.gz",
     "path":        buildDir.getAbsolutePath() + '/' + "openhab-${OHVersion}.tar.gz",
     "version":     "${OHVersion}",
     "release":     "${OHReleaseNumber}"
   ],[
-    'dist':        'openhab2-addons-release',
-    'description': 'Linux installation package for openHAB 2 addons.',
+    'dist':        'openhab-addons-release',
+    'description': 'Linux installation package for openHAB addons.',
     'debDist':     ['stable','testing','unstable'],
-    'packageName': 'openhab2-addons',
-    'url':         "https://bintray.com/openhab/mvn/download_file?file_path=org%2Fopenhab%2Fdistro%2Fopenhab-addons%2F${OHVersion}%2Fopenhab-addons-${OHVersion}.kar",
+    'packageName': 'openhab-addons',
+    'url':         "https://bintray.com/openhab/mvn/download_file?file_path=org/openhab/distro/openhab-addons/${OHVersion}/openhab-addons-${OHVersion}.kar",
     'path':        buildDir.getAbsolutePath() + '/' + "openhab-addons-${OHVersion}.kar",
     'version':     "${OHVersion}",
     'release':     "${OHReleaseNumber}"
   ],[
-    "dist":        'openhab2-testing',
-    "description": 'Linux installation package for openHAB 2.',
+    "dist":        'openhab-testing',
+    "description": 'Linux installation package for openHAB.',
     "debDist":     ['testing'],
-    "packageName": 'openhab2',
+    "packageName": 'openhab',
     "url":         "https://openhab.jfrog.io/openhab/libs-milestone-local/org/openhab/distro/openhab/${OHTestingVersion}/openhab-${OHTestingVersion}.tar.gz",
     "path":        buildDir.getAbsolutePath() + '/' + "openhab-${OHTestingVersion}.tar.gz",
     "version":     "${OHTestingVersion}",
     "release":     "${OHReleaseNumber}"
   ],[
-    'dist':        'openhab2-addons-testing',
-    'description': 'Linux installation package for openHAB 2 addons.',
+    'dist':        'openhab-addons-testing',
+    'description': 'Linux installation package for openHAB addons.',
```

Comment:
Should we call it "openHAB Add-ons" like in the `README.md` of the [repository](https://github.com/openhab/openhab-addons).